go-controller/pkg/ovn/gateway.go

```diff
@@ -834,7 +830,7 @@ func (gw *GatewayManager) gatewayInit(
 		}
 	}
 
-	gwRouter, err := gw.createGWRouter(l3GatewayConfig, gwLRPJoinIPs)
+	gwRouter, err := gw.createGWRouter(gwConfig.config, gwConfig.gwLRPJoinIPs)
```

Comment:
i dont like gwConfig.config. It's ambiguous. I think this is the config from unmarshaling annotations right? can we call it annoConfig or something more specific?